### PR TITLE
Add image upload and blog edit information

### PR DESCRIPTION
**Pull request by nathanshammah**
Add image upload and blog edit information to the readme. Include both ways an image can be added. Include information on typography in blog posts, including adding code snippets as a `gist`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,16 +30,50 @@ month: 2
...
+
 The Vercel automation will create a preview you can access by clicking the "Visit preview" link that shows up in the bot comment.
 If the changes look good, request a review by tagging one of the UF team members (e.g., `@nathanshammah`).
 
-### Adding images to a blog post
+## Adding images
+
+To add an image in a page, there are two options:
+
+1. (**No Code option**): Go to https://unitary-fund.vercel.app/author/image + copy the code (e..g, d394zg4utwqxynenamqf) + paste it in the markdown file as "::image[d394zg4utwqxynenamqf]". This option is the one required for adding the cover image to a new `event`.
```
Comment (natestemen) [inline README.md:74]: The `+` here reads to me as string concatenation. I'd rephrase as something like
```
Visit https://unitary.... and upload your desired image.
Copy the code it returns after upload and use such and such syntax to insert.
```

File: README.md [hunk 2]
```diff
@@ -30,16 +30,50 @@ month: 2
...
 
-To add an image in a document, upload the image to `public/images/`, and embed it into the desired page using the `![alt text](/images/your_image.png)` syntax.
+2. Upload the image under `public/images/` + embed image links in markdown files e.g., ![(images/image.png), and embed it into the desired page, e.g., using the `![alt text](/images/your_image.png)` syntax.
```
Comment (natestemen) [inline README.md:76]: I don't think the first syntax suggested in this change is correct.
```suggestion
2. Upload the image to `public/images/` and embed it into the desired page using the `![alt text](/images/your_image.png)` syntax.
```

File: README.md [hunk 3]
```diff
@@ -30,16 +30,50 @@ month: 2
...
+
+#### This is a heading in heading style four
+
+::gist[6cc14d37209de07abd707804f1b0219e]
+
+With slashes
+
+::gist[matt-lourens/6cc14d37209de07abd707804f1b0219e/]
+
+::gist[/matt-lourens//6cc14d37209de07abd707804f1b0219e/]
```
Comment (natestemen) [inline README.md:62]: I'd say we suggest a single one of these. Also it's unclear where `matt-lourens` is coming from . When uploading an image as described below, there is no user identifier that is embedded in the code. This either needs explanation, or removal.

File: README.md [hunk 4]
```diff
@@ -30,16 +30,50 @@ month: 2
...
+This is a standard paragraph with a [LINK](https://www.google.com). **This is bold paragraph text**
+
+- this is a list item
+- this is **bold** list item
+- this is a long list item which has lots of words in it and will probably break onto a second or even a third line so we can see how that looks at differnt sizes adn make sure it looks as it is intended too
+- this is a list item with a [LINK](https://www.google.com) or [two)](https://www.google.com) added in it. 
+
+### This is a heading in heading style three introducing a video
+
+https://www.youtube.com/watch?v=dB_3R84ewig
```
Comment (natestemen) [inline README.md:52]: I wonder if instead of demo-ing a subset of markdown here to act as a template, it might be more helpful to provide a link to more thorough documentation on how to use markdown.
- [GitHub docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
- [Write the Docs](https://www.writethedocs.org/guide/writing/markdown/)
- There's even a [markdown tutorial](https://www.markdowntutorial.com/)

**Review by natestemen — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (nathanshammah) [reply README.md:52]: For me it is useful to show an example, as there are some elements specific here  as far as I can tell, e.g., 
```
**This is a featured Paragraph it cannot contain bold text but can contain [links](https://www.unitary.fund).**
```
or 
```
::image[d394zg4utwqxynenamqf]
```
Even if standard markdown, these prove quite useful to me even if I spent time previously editing markdown, due to how the website works.

Also in favor of having a template file, but could iterate in a later version.

File: README.md (hunk 3 above)
Comment (nathanshammah) [reply README.md:62]: Good point. They seem to come from https://gist.github.com/matt-lourens/ and then the second is https://gist.github.com/matt-lourens/6cc14d37209de07abd707804f1b0219e.

Comment (nathanshammah) [inline README.md:62]:
```suggestion

## Including code snippets with gists
A supported way to include code snippets is by embedding gists. Gists are code snippets that you can create on Github. Make sure that they are public. For example, navigating to [gist.github.com/matt-lourens/](https://gist.github.com/matt-lourens/) one can click on one of the gists and obtain the url  https://gist.github.com/matt-lourens/6cc14d37209de07abd707804f1b0219e. In order to embed the related code in the blog post, you can do one of the following:

::gist[6cc14d37209de07abd707804f1b0219e]

or with slashes

::gist[matt-lourens/6cc14d37209de07abd707804f1b0219e/]
```

File: README.md [hunk 5]
```diff
@@ -30,16 +30,52 @@ month: 2
...
+
 The Vercel automation will create a preview you can access by clicking the "Visit preview" link that shows up in the bot comment.
 If the changes look good, request a review by tagging one of the UF team members (e.g., `@nathanshammah`).
 
-### Adding images to a blog post
+## Adding images
+
+To add an image in a page, there are two options:
+
+1. (**No Code option**): Go to https://unitary-fund.vercel.app/author/image + copy the code (e..g, d394zg4utwqxynenamqf) + paste it in the markdown file as "::image[d394zg4utwqxynenamqf]". This option is the one required for adding the cover image to a new `event`.
```
Comment (nathanshammah) [inline README.md:76]:
```suggestion
1. (**No Code option**): Go to https://unitary-fund.vercel.app/author/image and upload your desired image. 
 Copy the code it returns after upload and use such and such syntax to insert. (e.g, d394zg4utwqxynenamqf). Note that this option is the required one for adding a cover image to a new `event`.
```

**Comment by nathanshammah** [comment]
Thanks @natestemen! File updated, reads much better.

File: README.md [hunk 6]
```diff
@@ -30,16 +30,53 @@ month: 2
 year: 2023
 ---
 
-your markdown content
+# This is a heading style one
+
+**This is a featured Paragraph it cannot contain bold text but can contain [links](https://www.unitary.fund).**
```
Comment (natestemen) [inline README.md:35]: What flavor of markdown is this? Is a featured paragraph the same as a bolded paragraph?
```suggestion
**This is a featured paragraph it cannot contain bold text but can contain [links](https://www.unitary.fund).**
```

File: README.md [hunk 7]
```diff
@@ -30,16 +30,53 @@ month: 2
...
+
+More text
+
+::image[d394zg4utwqxynenamqf]
+
+## This is a heading in heading style two
+
+This is a standard paragraph with a [LINK](https://www.google.com). **This is bold paragraph text**
```
Comment (natestemen) [inline README.md:43]:
```suggestion
This is a standard paragraph with a [hyperlink](https://www.example.com). **This is bold paragraph text**.
```

File: README.md [hunk 8]
```diff
@@ -30,16 +30,53 @@ month: 2
...
+
+- this is a list item
+- this is **bold** list item
+- this is a long list item which has lots of words in it and will probably break onto a second or even a third line so we can see how that looks at differnt sizes adn make sure it looks as it is intended too
```
Comment (natestemen) [inline README.md:47]:
```suggestion
- this is a long list item which has lots of words in it and will probably break onto a second or even a third line so we can make sure it looks as it is intended too. Can even have multiple sentences within a list item.
```

File: README.md [hunk 9]
```diff
@@ -30,16 +30,53 @@ month: 2
...
+- this is a list item with a [LINK](https://www.google.com) or [two)](https://www.google.com) added in it.
```
Comment (natestemen) [inline README.md:48]:
```suggestion
- this is a list item with a [link](https://www.example.com) or [two](https://www.example.com) added in it. 
```

File: README.md [hunk 10]
```diff
@@ -30,16 +30,53 @@ month: 2
...
+This is a standard paragraph with a [LINK](https://www.google.com). **This is bold paragraph text**
+
+- this is a list item
+- this is **bold** list item
+- this is a long list item which has lots of words in it and will probably break onto a second or even a third line so we can see how that looks at differnt sizes adn make sure it looks as it is intended too
+- this is a list item with a [LINK](https://www.google.com) or [two)](https://www.google.com) added in it. 
+
+### This is a heading in heading style three introducing a video
+
+https://www.youtube.com/watch?v=dB_3R84ewig
```
Comment (natestemen) [inline README.md:52]: Is this intended to be an embedded video or just a link? I'm not confident this will render as a clickable link (and I can't check as we do not have dev docs yet 😕).

File: README.md [hunk 11]
```diff
@@ -30,16 +30,53 @@ month: 2
...
+
+#### This is a heading in heading style four
+
+
+## Including code snippets with gists
+A supported way to include code snippets is by embedding gists. Gists are code snippets that you can create on Github. Make sure that they are public. For example, navigating to [gist.github.com/matt-lourens/](https://gist.github.com/matt-lourens/) one can click on one of the gists and obtain the url  https://gist.github.com/matt-lourens/6cc14d37209de07abd707804f1b0219e. In order to embed the related code in the blog post, you can do one of the following:
```
Comment (natestemen) [inline README.md:58]:
```suggestion
A supported way to include code snippets is by embedding gists.
Gists are code snippets that you can create on Github.
Learn how to create one [here](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists), and don't forget to ensure it is public!
Once created, use the hash from the gist URL with the syntax below.
```

File: README.md [hunk 12]
```diff
@@ -30,16 +30,53 @@ month: 2
...
+
+::gist[6cc14d37209de07abd707804f1b0219e]
+
+or with slashes
```
Comment (natestemen) [inline README.md:62]:
```suggestion
```
Let's save folks the decision and just provide one option.

File: README.md [hunk 13]
```diff
@@ -30,16 +30,53 @@ month: 2
...
 The Vercel automation will create a preview you can access by clicking the "Visit preview" link that shows up in the bot comment.
 If the changes look good, request a review by tagging one of the UF team members (e.g., `@nathanshammah`).
 
-### Adding images to a blog post
+## Adding images
+
+To add an image in a page, there are two options:
+
+1. (**No Code option**): Go to https://unitary-fund.vercel.app/author/image and upload your desired image. 
+ Copy the code it returns after upload and use such and such syntax to insert. (e.g, d394zg4utwqxynenamqf). Note that this option is the required one for adding a cover image to a new `event`.
```
Comment (natestemen) [inline README.md:77]: I think you may have copy and pasted my suggestion from https://github.com/unitaryfund/unitary.fund/pull/359#discussion_r1316369594 which was a template, but did not include all the details (which is why I did not use the suggestion tool). E.g. "such and such syntax" needs replaced.

**Review by natestemen — CHANGES_REQUESTED** [review]

File: README.md [hunk 14]
```diff
@@ -30,16 +30,53 @@ month: 2
...
+::image[d394zg4utwqxynenamqf]
+
+## This is a heading in heading style two
+
+This is a standard paragraph with a [hyperlink](https://www.example.com). **This is bold paragraph text**.
+
+- this is a list item
+- this is **bold** list item
+- this is a long list item which has lots of words in it and will probably break onto a second or even a third line so we can make sure it looks as it is intended too. Can even have multiple sentences within a list item.
+- this is a list item with a [LINK](https://www.google.com) or [two)](https://www.google.com) added in it.
```
Comment (nathanshammah) [inline README.md:48]:
```suggestion
- this is a list item with a [link](https://unitaryhack.dev/) added in it. 
```

File: README.md (hunk 10 above)
Comment (nathanshammah) [reply README.md:52]: @natestemen yeah it's strange but it should render as an embedded frame. I just tested it [here](https://unitary-fund-prel-lj20ttudw-unitary-fund.vercel.app/posts/test) in [this PR](https://github.com/unitaryfund/unitary.fund/pull/367). I believe it is a special feature implemented by the designers, not a general property of md. 

I think the same applies for paragraph formatting, that is why I suggest keeping reference to it.

File: README.md [hunk 15]
```diff
@@ -30,16 +30,53 @@ month: 2
...
 The Vercel automation will create a preview you can access by clicking the "Visit preview" link that shows up in the bot comment.
 If the changes look good, request a review by tagging one of the UF team members (e.g., `@nathanshammah`).
 
-### Adding images to a blog post
+## Adding images
+
+To add an image in a page, there are two options:
+
+1. (**No Code option**): Go to https://unitary-fund.vercel.app/author/image and upload your desired image. 
+ Copy the code it returns after upload and use such and such syntax to insert. (e.g, d394zg4utwqxynenamqf). Note that this option is the required one for adding a cover image to a new `event`.
```
Comment (nathanshammah) [inline README.md:77]:
```suggestion
 Copy the code it returns after upload, e.g, d394zg4utwqxynenamqf, and use the syntax `::image[d394zg4utwqxynenamqf]` to insert it. Note that this option is the required one for adding a cover image to a new `event`.
```

**Comment by nathanshammah** [comment]
Thanks @natestemen. Regarding the YT embedding: I just tested it [here](https://unitary-fund-prel-lj20ttudw-unitary-fund.vercel.app/posts/test) from this PR https://github.com/unitaryfund/unitary.fund/pull/367. I believe it is a special feature implemented by the designers, not a general property of md. (I think the same applies for paragraph formatting, that is why I suggest keeping reference to it.)

If it looks good please merge directly as there is a guest blog post that would benefit from this information (#364). (We can open issues to further refine the guidelines).

**Review by natestemen — APPROVED** [review]
